Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderWorker.cpp

```diff
@@ -152,7 +113,58 @@ namespace ScriptCanvasBuilder
             }
         }
 
-        m_processEditorAssetDependencies.clear();
+        AZ::SerializeContext* serializeContext{};
+        AZ::ComponentApplicationBus::BroadcastResult(serializeContext, &AZ::ComponentApplicationBus::Events::GetSerializeContext);
+        AZ_Assert(serializeContext, "SerializeContext is required to enumerate dependent assets in the ScriptCanvas file");
+
+        AZStd::unordered_multimap<AZStd::string, AssetBuilderSDK::SourceFileDependency> jobDependenciesByKey;
+        auto assetFilter = [this, &jobDependenciesByKey]
+            ( void* instancePointer
+            , const AZ::SerializeContext::ClassData* classData
+            , [[maybe_unused]] const AZ::SerializeContext::ClassElement* classElement)
+        {
+            auto azTypeId = classData->m_azRtti->GetTypeId();
+            
+            if (azTypeId == azrtti_typeid<AZ::Data::Asset<ScriptCanvas::SubgraphInterfaceAsset>>())
+            {
+                const auto* subgraphAsset = reinterpret_cast<AZ::Data::Asset<const ScriptCanvas::SubgraphInterfaceAsset>*>(instancePointer);
```

Comment:
replace auto* with just auto as the type T* is already present on the right of the expression